sql/2020/11_Security/cryptominer_share.sql

```diff
@@ -0,0 +1,38 @@
+#standardSQL
+--share of cryptominers
+WITH
+  total AS (
+  SELECT
+    COUNT(0) AS ttl
+  FROM
+    `httparchive.technologies.2020_10_01_desktop`
```

Comment:
Why October?

```suggestion
    `httparchive.technologies.2020_08_01_desktop`
```

Can we also not inline this query like we usually do instead of a separate WITH query?